crates/edr_napi/src/result.rs

```diff
@@ -113,7 +113,7 @@ impl From<edr_evm::HaltReason> for ExceptionalHalt {
         match halt {
             edr_evm::HaltReason::OutOfGas(..) => ExceptionalHalt::OutOfGas,
             edr_evm::HaltReason::OpcodeNotFound => ExceptionalHalt::OpcodeNotFound,
-            edr_evm::HaltReason::InvalidEFOpcode => ExceptionalHalt::InvalidEFOpcode,
+            edr_evm::HaltReason::InvalidEFOpcode => ExceptionalHalt::InvalidFEOpcode,
```

Comment:
Should `edr_evm::HaltReason::InvalidEFOpcode` be renamed as well?

Reply:
The `edr_evm::HaltReason::InvalidEFOpcode` is a re-export from `revm`. They renamed it, released, and then [reverted it](https://github.com/bluealloy/revm/commit/81818a9fd088de7a3df65b9f9a70a180d5a0fe79) based on feedback from Franco.

As they haven't done a new release, that change will have to wait until `revm` releases. I just wanted to affecting the `index.d.ts` bindings unnecessarily.

Reply:
As part of the refactor to remove "const generic bool" I had to upgrade to the latest revm, which also resolves this comment.